src/Components/MovieCard.js

```diff
@@ -7,13 +7,13 @@ const MovieCard = (props) => {
     if (findMovieRating) {
       userRate = <section>User: {findMovieRating.rating}</section>
     } else {
-      userRate = <section>RATE ME</section>
+      userRate = <button>RATE ME</button>
     }
```

Comment:
I remember suggesting this button, but we could possibly do a "Rate Me" label of sorts that invites the user to click on movie-card...rather than a button that would lead to the same page to do the same.